pkg/apis/kubeone/types.go

```diff
@@ -301,6 +303,15 @@ type CanalSpec struct {
 	MTU int `json:"mtu,omitempty"`
 }
 
+// CiliumSpec defines the Cilium CNI plugin
+type CiliumSpec struct {
+	// Disable kube-proxy to use Cilium eBPF replacement
+	// default value is false
+	DisableKubeProxy bool `json:"disableKubeProxy,omitempty"`
+	// Weather to deploy Hubble relay and UI
```

Comment:
Please make this a valid Go comment.

```suggestion
	// EnableHubble is used to deploy Hubble relay and UI
```